From c97a6e92481f283e3f5eaa0033a1fa54a23cc924 Mon Sep 17 00:00:00 2001
From: Bruce Forstall <brucefo@microsoft.com>
Date: Mon, 11 Dec 2017 17:53:25 -0800
Subject: [PATCH] Up pri-1 timeouts again

180 minutes was still not enough; we were still getting timeouts.
Up it to 240 minutes.
---
 netci.groovy | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/netci.groovy b/netci.groovy
index 5c21777..28b0c38 100755
--- a/netci.groovy
+++ b/netci.groovy
@@ -414,7 +414,7 @@ def static setTestJobTimeOut(newJob, isPR, architecture, scenario) {
     else if (!(scenario == 'default' && isPR == true)) {
         // Pri-1 test builds take a long time. Default PR jobs are Pri-0; everything else is Pri-1
         // (see calculateBuildCommands()). So up the Pri-1 build jobs timeout.
-        Utilities.setJobTimeout(newJob, 180)
+        Utilities.setJobTimeout(newJob, 240)
     }
     // Non-test jobs use the default timeout value.
 }
-- 
2.7.4

